### PR TITLE
Chore: cumulative milestone calculations

### PR DESCRIPTION
**Pull request by dleard**
Adds computed columns for the following cumulative calculations:
- eligible_expenses
- gross_payments
- net_payments
- holdback_amount

~~This should unblock #984~~ 984 has been dropped. The functions have been reworked to be used in #717

### CODE REVIEW

File: schema/deploy/computed_columns/form_change_eligible_expenses_to_date.sql
Comment (pbastia) [inline schema/deploy/computed_columns/form_change_eligible_expenses_to_date.sql:6]: this function will return the same value for all milestones, should we only sum the milestones that have been submitted before / including this milestone?

Comment (dleard) [reply schema/deploy/computed_columns/form_change_eligible_expenses_to_date.sql:6]: These functions are no longer calculating these values for individual milestones. They are for the budgets, payments and expenses page. So they're meant to be a total sum of all milestones.

File: schema/deploy/computed_columns/form_change_gross_payments_to_date.sql
Comment (pbastia) [inline schema/deploy/computed_columns/form_change_gross_payments_to_date.sql:11]: I wonder if in the `payments_to_date` we should include the maximum amounts, which are payments that haven't been made yet

Comment (dleard) [reply schema/deploy/computed_columns/form_change_gross_payments_to_date.sql:11]: This is what I understood from talking with @nanyangpro but we can wait for confirmation before merging

Comment (dleard) [reply schema/deploy/computed_columns/form_change_gross_payments_to_date.sql]: You're right. We are not to include the maximum amounts here, only completed milestones (ones with adjusted/calculated values) are to be counted. Fixed.

File: schema/test/unit/computed_columns/form_change_eligible_expenses_to_date_test.sql
Comment (pbastia) [inline schema/test/unit/computed_columns/form_change_eligible_expenses_to_date_test.sql:74]: We might want to add one more milestone where `totalEligibleExpenses` is not set in the test suite, to make sure it doesn't break

**Review by pbastia — CHANGES_REQUESTED** [review]
Just one minor test case! This is great 💯

**Review by pbastia — APPROVED** [review]

**Review by pbastia — APPROVED** [review]

**Pull request merged**
